posts/2022-05-28-liberty-deep-dive.adoc

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Introducing a hands-on technical deep dive on Liberty!"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Introducing a hands-on technical deep dive on Liberty! - OpenLiberty.io
+seo-description: The Liberty Deep Dive will teach you how to build a RESTful microservice on Liberty with Jakarta EE and MicroProfile, document your REST API, persist data to a database, secure your applications using JSON Web Tokens (JWT), test your applications and more.
+blog_description: "TThe Liberty Deep Dive will teach you how to build a RESTful microservice on Liberty with Jakarta EE and MicroProfile, document your REST API, persist data to a database, secure your applications using JSON Web Tokens (JWT), test your applications and more."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Introducing a hands-on technical deep dive on Liberty!
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+We are excited to announce the release of our newest tutorial, A Technical Deep Dive on Liberty! Our team have been hard at work to produce a single tutorial which covers many cloud-native concepts and bundles them into one application. This tutorial is aimed to help developers, from Liberty novices to those more experienced, develop, build, test, and deploy cloud-native Java applications and microservices on Open Liberty. You will learn how to build a RESTful microservice on Liberty with Jakarta EE and MicroProfile, documenting your REST API, persisting data to a database, securing your applications using JSON Web Tokens (JWT), testing your applications and more. You will use Maven throughout this exercise to build the microservice and to interact with the running Liberty instance. Lastly, you’ll build a container image for the microservice and deploy it to Kubernetes using the Open Liberty Operator.
```

Comment:
```suggestion
We are excited to announce the release of our newest tutorial, A Technical Deep Dive on Liberty!

Our team has been hard at work to produce a single end-to-end tutorial that covers important cloud-native concepts, developing and deploying applications and microservices using open cloud-native Java APIs in Jakarta EE and MicroProfile while following best practices for working with Liberty. This tutorial is aimed to help developers, from Liberty novices to those more experienced, develop, build, test, and deploy your applications on Liberty running in Kubernetes and clouds.

In this tutorial, you will learn how to build RESTful microservice on Liberty with Jakarta EE and MicroProfile, documenting your REST API, externalizing configuration, persisting data to a database, securing your application using JSON Web Tokens (JWT), adding health checks, providing metrics, testing your applications and more. You will also look at building container image for your Java microservice and deploying it to Kubernetes using the Open Liberty Operator.
```

Reply:
```suggestion
We are excited to announce the release of our newest tutorial, A Technical Deep Dive on Liberty!

Our team has been hard at work to produce a single end-to-end tutorial that covers developing and deploying applications and microservices by using open cloud-native Java APIs in Jakarta EE and MicroProfile while following best practices for working with Liberty. This tutorial is aimed to help developers, from Liberty novices to those more experienced, develop, build, test, and deploy your applications on Liberty running in Kubernetes and clouds.

In this tutorial, you will learn how to:
- build a RESTful microservice on Liberty with Jakarta EE and MicroProfile 
- document your REST API 
- externalize your configuration
- persist data to a database 
- secure your application using JSON Web Tokens (JWT) 
- add health checks 
- provide metrics 
- test your applications,  and more. 

You will also build a container image for your Java microservice and deploy it to Kubernetes by using the Open Liberty Operator.
```

Reply:
While I appreciate that we _are_ excited to get this published after all our hard work, I think we're centering ourselves a bit much here. I doubt the reader cares about our excitement and hard work so much as what the tutorial will do for them.

Keep the "We are excited..." sentence if you like, but lose the exclamation mark/point (!).

Then start the next paragraph with "A Technical Deep Dive on Liberty is an end-to-end tutorial that walks you through how to develop and deploy cloud-native Java applications in Jakarta EE and MicroProfile. The tutorial is aimed at Java developers, from Liberty novices to those more experienced with writing applications to run on Liberty."

Then change the lead-in sentence to: "In A Technical Deep Dive on Liberty, you will learn how to:" (then have the bulleted list like David suggests as it's nice and easy to skim down.

Shouldn't that last sentence also be bullets rather than a separate sentence? So:

- build a container image for your Java microservice
- deploy the container image to Kubernetes using the Open Liberty Operator